assets/operator/ocp-aarch64/ruby/imagestreams/ruby-rhel-aarch64.json

```diff
@@ -34,27 +34,6 @@
 					"type": "Local"
 				}
 			},
-			{
```

Comment:
This is gonna break a lot of stuff:
* rails samples depend on this, e.g.: https://github.com/openshift/cluster-samples-operator/blob/main/assets/operator/ocp-x86_64/rails/templates/rails-pgsql-persistent.json#L534 (which is also referenced in `openshift/origin` https://github.com/openshift/origin/blob/main/test/extended/image_ecosystem/s2i_ruby.go#L16, https://github.com/openshift/origin/blob/main/test/extended/image_ecosystem/sample_repos.go#L143 and possibly elsewhere.
* This ruby version is also used in the `openshift/origin` tests like https://github.com/openshift/origin/blob/main/test/extended/image_ecosystem/s2i_images.go#L36